src/index/api.rs

```diff
@@ -1,11 +1,19 @@
 use crate::{error::Result, utils::MetadataPackage};
 use chrono::{DateTime, Utc};
 use failure::err_msg;
-use reqwest::{header::ACCEPT, Client};
+use reqwest::header::{HeaderValue, ACCEPT, USER_AGENT};
 use semver::Version;
 use serde_json::Value;
 use std::io::Read;
 
+const APP_USER_AGENT: &str = concat!(
+    env!("CARGO_PKG_NAME"),
+    "/",
+    env!("CARGO_PKG_VERSION"),
```

Comment:
I don't think including the version is useful, we never bump it and it doesn't really have a meaning nowadays.

Reply:
Using a user agent from an env var is probably better since it'll mean that only *our* instance of docs.rs has the agent instead of every single build (So we can be uniquely identified)

Reply:
That's not the purpose of a user-agent, it's to know what software is making the request. The IP from the crates.io logs can be used for checking the instance.